src/deals/list/deals.ts

```diff
@@ -174,14 +174,10 @@ export class Deals {
     this.showMine = !this.showMine;
     if (this.showMine) {
       //if showing only "my deals" check to see which tab to display by default if there are no deals in either tab
-      const openDeals = this.isTabVisible(0, this.showMine, this.ethereumService.defaultAccountAddress);
-      const partneredDeals = this.isTabVisible(1, this.showMine, this.ethereumService.defaultAccountAddress);
-      if (openDeals) {
+      const hasPartneredDeals = this.isTabVisible(1, this.showMine, this.ethereumService.defaultAccountAddress);
+      if (!hasPartneredDeals) {
         this.cardIndex = 0;
       }
```

Comment:
Can you give a little background of why the "openDeals" case is not needed anymore? (because default?)

Reply:
You are right. I tested more and it seems I have to add the other condition back. I also fixed another old one we had when the account was changed in metamask. (check the removed `async` for that)